content/chainguard/libraries/java/build-configuration.md

```diff
@@ -416,14 +416,132 @@ allprojects {
 }
 ```
 
+## Bazel
+
+[Bazel](https://bazel.build/) is a fast, scalable, and extensible build tool
+commonly used in large-scale projects.
+
+### Remove Bazel Caches
+
+Bazel uses a cache to store downloaded artifacts. When adopting Chainguard
+Libraries for Java, you must delete this cache to ensure that libraries are
+downloaded again. By default, the cache is located in the `.cache/bazel` on
+Linux or `/private/var/tmp/_bazel_$USER` on MacOS directory in your user's home
+directory. Use the following command to delete it:
+
+```shell
+bazel clean --expunge
+```
+
+The [Bazel documentation on output
+directories](https://bazel.build/remote/output-directories) contains further
+details.
+
+### Change Bazel Configuration
+
+Before running a new build, you must configure access to the Chainguard
+Libraries for Java. If the administrator for your organization’s repository
+manager created a new repository or virtual repository, you must update your
+Bazel configuration to use the repository manager.
+
+Bazel uses `MODULE.bazel` files to define external dependencies as `artifacts`.
+You can configure a Maven repository for artifact retrieval using `repositories`
+from the
+[rules_jvm_external](https://github.com/bazel-contrib/rules_jvm_external) rule: 
+
+Following is an example configuration for a repository manager:
+
+```MODULE.bazel
```

Comment:
Not sure about using this.. or something like Go template (is it?) .. or leave empty (then is uses Output .. which is also kinda bad) .. asked on team slack channel

Reply:
I think it's fine for now

Reply:
That was my hunch here as well.. Thanks for confirming @ltagliaferri